test/README.md

```diff
@@ -13,6 +13,16 @@ It is using the [Godog](https://github.com/cucumber/godog) framework.
 
 `../hack/run-tests.sh --feature {FEATURE}`
 
+Example:
+
+`../hack/run-tests.sh --feature "features/install_kogitoinfra.feature`
+
+### Run using your own Kogito Operator repository
+
+`../hack/run-tests.sh --operator_image quay.io/jcarvaja/kogito-cloud-operator`
```

Comment:
maybe add the `operator_tag`. Many people are making the mistake because both are separated (image name and tag) 
if you built your own, you may have also changed the tag.

Reply:
Added